index.js

```diff
@@ -121,6 +159,11 @@ function updatemodalTitle(newTitle) {
   modalTitle.textContent = newTitle;
 }
 
+function UpdateShoppingtitle(newTitle) {
```

Comment:
Los nombres de funciones van con minúscula inicial y cada `nueva palabra` inicia en mayúscula
```suggestion
function updateShoppingTitle(newTitle) {
```